src/renderer/views/account/AccountView.tsx

```diff
@@ -5,41 +5,40 @@ import { Container, InputLabel } from "@material-ui/core";
 import CreateAccountView from "./CreateAccountView";
 import RevokeAccountView from "./RevokeAccountView";
 
-import { useLocale } from "../../i18n";
+import { T } from "@transifex/react";
 import { Account } from "../../../interfaces/i18n";
 
 const AccountView: React.FC<IStoreContainer> = observer(
   ({ accountStore, routerStore }) => {
-    const { locale } = useLocale<Account>("account");
     return (
       <div>
         <button
           onClick={() => {
             routerStore.push("/account/create");
           }}
         >
-          {locale("키 생성하기")}
+          <T _str="create key" _tags="account" />
```

Comment:
IMO use variables instead of a scala-type string are better.

Reply:
You mean for every tag initialization?

Reply:
Yes, like this
```suggestion
          <T _str="create key" _tags=`${transifexTag}` />
```

Reply:
Seems like nice idea to me! But Transifex CLI's const lookup doesn't work on some APIs, so I filed a PR to resolve this issue. Hopefully they'll merge it and we could use them.